src/test/java/th/ac/kmitl/science/comsci/example/models/SellerTest.java

```diff
@@ -0,0 +1,26 @@
+package th.ac.kmitl.science.comsci.example.models;
+
+import junit.framework.Assert;
+import org.junit.Test;
+
+public class SellerTest {
+
+    @Test
+    public void testInputSeller(){
+        String id = "id58050276";
+        String globalId = "GId58050276";
+        String name = "thanathat office";
+        String taxId = "TId58050276";
+        Seller seller = new Seller();
```

Comment:
Add one blank line to separate declaration point, processing point and validation (or return) point